app/assets/stylesheets/components/_expander.scss

```diff
@@ -39,6 +39,7 @@
   width: 30px;
   height: 40px;
   fill: govuk-colour("black");
+  padding-left: 3.5px;
```

Comment:
we can adjust the `left` property and then we don't seen the padding declaration. 
see the comment below about half pixels

Reply:
> we can adjust the left property and then we don't seen the padding declaration.

removing `padding-left: 3px` and changing `left` from 6 to 9px should produce the same visual result

Reply:
Sorry, I pushed the wrong commit when I resolved these, it should be better now